Doctrine/Phpcr/MediaManager.php

```diff
@@ -133,8 +134,12 @@ public function setDefaults(MediaInterface $media, $parentPath = null)
      */
     public function mapPathToId($path, $rootPath = null)
     {
-        // The path is being the id
-        $id = PathHelper::absolutizePath($path, '/');
+        try {
+            // The path is being the id
+            $id = PathHelper::absolutizePath($path, '/');
+        } catch (RepositoryException $e) {
+            throw new \OutOfBoundsException($e->getMessage(), 0, $e);
```

Comment:
Ok, but in this case we have to catch this exception (and the ``\OutOfBoundException`` one) in the ImageController : https://github.com/symfony-cmf/MediaBundle/blob/master/Controller/ImageController.php#L32

Currently, the ``MediaManagerInterface::mapUrlSafePathToId`` is only supposed to throw an ``\OutOfBoundException``.

Does it seem ok to you ?